### PR TITLE
SITL: allow generation of vibration with exact frequencies

### PR DESCRIPTION
**Pull request by tridge**
This is useful for testing FFT on logs

### CODE REVIEW

**Comment by rmackay9** [comment]
very cool!

File: libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp [hunk 1]
```diff
@@ -119,9 +130,17 @@ void AP_InertialSensor_SITL::generate_gyro(uint8_t instance)
     float q = radians(sitl->state.pitchRate) + gyro_drift();
     float r = radians(sitl->state.yawRate) + gyro_drift();
 
-    p += gyro_noise * rand_float();
-    q += gyro_noise * rand_float();
-    r += gyro_noise * rand_float();
+    const Vector3f &vibe_freq = sitl->vibe_freq;
```
Comment (magicrub) [inline libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp]: why is vibe_freq a member variable in SITL.h and a local variable reference?

Comment (tridge) [reply libraries/AP_InertialSensor/AP_InertialSensor_SITL.cpp]: it is a convenient alias, to make the code neater. It converts from AP_Vector3f to Vector3f, and makes the following code easier to read

**Comment by magicrub** [comment]
I disagree. Using the same name is confusing. Would rather it see with an underscore or something else just so it's obvious that it's different

**Pull request merged**